<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerWindow</class>
 <widget class="QMainWindow" name="WorkerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="customer_tab_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="catalog_tab">
     <attribute name="title">
      <string>Catalog</string>
     </attribute>
     <widget class="QTableWidget" name="catalog_table_widget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>521</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>UPC Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prime price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Retail price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Discount price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount on stock</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="orders_tab">
     <attribute name="title">
      <string>Orders</string>
     </attribute>
     <widget class="QComboBox" name="status_filter_comboBox">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="status_filter_label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter by status:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="orders_table_widget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>571</width>
        <height>521</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Order number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cost</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="details_table_widget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>130</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Product Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cost</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="order_number_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Details of Order: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="hide_details_button">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Hide</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Statisctics</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="account_tab">
     <attribute name="title">
      <string>My account</string>
     </attribute>
     <widget class="QLineEdit" name="username_line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="email_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="telephone_number_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telephone number</string>
      </property>
     </widget>
     <widget class="QLabel" name="address_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="telephone_number_line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="address_line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>271</width>
        <height>71</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="change_telephone_number_button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QPushButton" name="change_address_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="invalid_telephone_number_label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Invalid telephone number, try again</string>
      </property>
     </widget>
     <widget class="QLabel" name="invalid_address_label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Invalid address, try again</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="signed_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>You are signed as</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_out_button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign out</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
